tools/shell/tests/test_shell_basics.py

```diff
@@ -550,14 +588,26 @@ def test_output_csv_mode(shell, random_filepath):
     result.stdout = open(random_filepath, 'rb').read()
     result.check_stdout(b'42')
 
-def test_issue_6204(shell):
+def test_issue_6204(shell, random_filepath):
     test = (
         ShellTest(shell)
-        .statement(".output foo.txt")
+        .statement(f".output {random_filepath.as_posix()}")
         .statement("select * from range(2049);")
     )
     result = test.run()
-    result.check_stdout("")
+    result.check_stdout(None)
+
+    with open(random_filepath, 'r', encoding='utf-8') as f:
+        lines = f.readlines()
+        # The 5 extra lines are from the header and duckbox
+        assert len(lines) == 2049 + 2 + 3 # range + header + duckbox
+        assert lines[0] == '┌───────┐\n'
```

Comment:
We prefer not to match too exactly to formatting - maybe we can just match a few of the result values instead? Or verify all result values are in there (i.e. run a loop from 0...2048)?